string mode
string srcPath
string desPath
---
bool result